rental.rb

```diff
@@ -0,0 +1,12 @@
+require 'date'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book
+
+  def initialize(book, person)
+    @date = DateTime.new
+    @book = book
+    @person = person
+  end
+end
```

Comment:
I have initialized it under constuctor already. But passing it as an argument which you suggest isn't an activity requirement. It might be an optional suggestion and still it wont necessitate the removal of `date` library.